Userland/Libraries/LibJS/Runtime/Error.cpp

```diff
@@ -48,6 +55,26 @@ ThrowCompletionOr<void> Error::install_error_cause(Value options)
     return {};
 }
 
+void Error::populate_stack()
+{
+    AK::StringBuilder stack_string_builder {};
+
+    // Note: We roughly follow Firefox' formatting
+    // FIXME: Firefox omits the first stack frame when it is a call to the ErrorContructor
+    // FIXME: Chrome prepends the ClassName and the error.message
+    // We dont want to capture the global exectution context, so we omit the last frame
+    for (size_t i = vm().execution_context_stack().size() - 1; i > 0; --i) {
+        auto const* frame = vm().execution_context_stack()[i];
+        if (auto const* current_node = frame->current_node)
+            stack_string_builder.appendff("{} at {}:{}:{}\n", frame->function_name, current_node->source_range().filename, current_node->source_range().start.line, current_node->source_range().start.column);
```

Comment:
Although the problem was getting valid `current_node`s there are actually valid execution contexts which don't have an associated scriptormodule and thus no `current_node` so we probably want a check and then just print `func at <unknown>` or something.